.env.telostestnet

```diff
@@ -68,4 +69,5 @@ REACT_APP_EOS_CHAIN_ID=1eaa0824707c8c16bd25145493bf062aecddfeb56c736f6ba6397f319
 REACT_APP_EOS_USE_BP_JSON_ON_CHAIN=false
 REACT_APP_EOS_BP_JSON_ON_CHAIN_CONTRACT=producerjson
 REACT_APP_EOS_BP_JSON_ON_CHAIN_TABLE=producerjson
-REACT_APP_EOS_BP_JSON_ON_CHAIN_SCOPE=producerjson
\ No newline at end of file
+REACT_APP_EOS_BP_JSON_ON_CHAIN_SCOPE=producerjson
+REACT_APP_TOKEN_SYMBOL=TLOS
```

Comment:
@JustinCast I think that this new environment variable should be added to kubernetes configmap and push files

Reply:
also it should be in the dockerfile

Reply:
@adriexnet thanks for the observations. Fixed!